src/common/types/store-data/unified-data-interface.ts

```diff
@@ -45,8 +45,9 @@ export interface InstancePropertyBag {
 export type StoredInstancePropertyBag = InstancePropertyBag;
 
 export interface UnifiedResult {
-    id: string;
+    uid: string;
     status: ResultStatus;
+    ruleId: string;
     evaluation: StoredInstancePropertyBag;
```

Comment:
We should remove the evaluation property as it will now be redundant/not necessary.